docs/learn/learn-guides-DOT-KSM-bridge.md

```diff
@@ -126,3 +126,23 @@ balances of any account on Kusama Asset Hub can be queried on-chain through the
 `foreignAssets`pallet as shown below.
 
 ![Wrapped KSM Balance](../assets/bridge-hub/PAH-KSM-Balance.png)
+
+## Transfer of arbitrary assets between Polkadot Asset Hub and Kusama Asset Hub
+
+Transfer functionality of assets other than DOT and KSM between Asset Hubs is yet to be enabled. Once this
+functionality is enabled, assets which are sufficient or non-sufficient on the Asset Hubs can be bridged. 
+
+:::caution Avoid Asset Traps
+
+To avoid issues on the receiving side for non-sufficient assets, make sure to call
```

Comment:
Can we add more clarity here? Let's say I'm transferring an asset from account A on Polkadot Asset Hub to the same account on Kusama Asset Hub which does not have the ED. On which chain should I make this call?

What if I'm sending it to account B on Kusama Asset Hub that I do not own, can I make this call for it?

Reply:
> On which chain should I make this call?

Always on the recipient account of the destination chain. That is the one that will not accept the incoming assets if either:
- account does not exist yet and incoming assets do not satisfy creation ED,
- account already satisfies ED but has reached its "maximum number of different assets" limit.

A successful `pallet-assets::touch` will eliminate both preconditions above, and guarantees a subsequent incoming transfer _of the same asset_ that has been "touched", will also be successful.

> What if I'm sending it to account B on Kusama Asset Hub that I do not own, can I make this call for it?

Yes, there is [pallet-assets::touch-other()](https://github.com/paritytech/polkadot-sdk/blob/0ef37c75401b78b61ed35ce27af8b964da27bb3c/substrate/frame/assets/src/lib.rs#L1616) for this case.

Reply:
Thank you!